docs/channel/README.md

```diff
@@ -20,13 +16,8 @@ The dataplane is designed in a way that it only communicates the control plane v
 
 ## Comparison with other KafkaChannel implementations
 
-You may find a comparison of this new KafkaChannel implementation with
-the [consolidated KafkaChannel](https://github.com/knative-extensions/eventing-kafka/blob/main/pkg/channel/consolidated/README.md)
-and
-the [distributed KafkaChannel](https://github.com/knative-extensions/eventing-kafka/blob/main/pkg/channel/distributed/README.md)
-implementations.
```

Comment:
I'd prefer to keep these lines (with links to the appropriate permalinks to commits before these files were removed), and just mention that those channel implementations are deprecated.

Otherwise, we can just remove the whole table. 

I think that the mentions of the other implementations in the table are confusing without the context of these lines though.

Reply:
Yes, I agree with you. I added back those lines with some changes.